### PR TITLE
Resolve the Attendees Column on Events

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#88047](http://central.tri.be/issues/88047) [C#92287](http://central.tri.be/issues/92287)
---

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $deprecated is never used | [Tickets.php:137](https://github.com/moderntribe/event-tickets/blob/fb110c04a2d4d422d711852f903540021c49c01f/src/Tribe/Admin/Columns/Tickets.php#L137)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Admin/Columns/Tickets.php [hunk 1]
```diff
@@ -105,11 +105,14 @@ protected function render_tickets_entry( $post_id ) {
...
 	 * Iterates over an array of tickets to fetch the sale total.
 	 *
+	 * @deprecated TBD
+	 *
 	 * @param Tribe__Tickets__Ticket_Object[] $tickets
 	 *
 	 * @return int The sale total.
 	 */
 	protected function get_sold( $tickets ) {
+		_deprecated_function( __METHOD__, 'TBD', 'tribe( "tickets.handler" )->get_ticket_totals()' );
```
Comment (zbtirrell) [inline src/Tribe/Admin/Columns/Tickets.php]: why are we flagging this as deprecated, but still using it?

Comment (bordoni) [reply src/Tribe/Admin/Columns/Tickets.php]: My bad, Oversight.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $deprecated is never used | [Tickets.php:138](https://github.com/moderntribe/event-tickets/blob/f9ff0e16de408c0839b094cc446f95a2968abdfd/src/Tribe/Admin/Columns/Tickets.php#L138)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $deprecated is never used | [Tickets.php:138](https://github.com/moderntribe/event-tickets/blob/fdd6a036b3ff55e26b97ca11528710fc5940b2fb/src/Tribe/Admin/Columns/Tickets.php#L138)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $deprecated is never used | [Tickets.php:138](https://github.com/moderntribe/event-tickets/blob/dd2fdbc4f0e86cff704db15a3602e9706325c75c/src/Tribe/Admin/Columns/Tickets.php#L138)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Admin/Columns/Tickets.php [hunk 2]
```diff
@@ -89,27 +89,31 @@ public function render_column( $column, $post_id ) {
...
-		$output = '—';
+		$output = '&mdash;';
+		$post = get_post( $post_id );
+		$totals = tribe( 'tickets.handler' )->get_post_totals( $post );
 
-		$tickets = Tribe__Tickets__Tickets::get_all_event_tickets( $post_id );
-		if ( empty( $tickets ) ) {
-			return '—';
+		if ( 0 === $totals['tickets'] ) {
+			return $output;
```
Comment (lucatume) [inline src/Tribe/Admin/Columns/Tickets.php]: Why set the `$output` var to `-` and *then* return it only in this case? Do not set `$output` and just return `$mdash;`

File: src/Tribe/Admin/Columns/Tickets.php [hunk 3]
```diff
@@ -121,51 +125,32 @@ protected function get_sold( $tickets ) {
...
 	 */
-	protected function get_percentage_string( $post_id, array $tickets = array() ) {
-		$sold  = 0;
-		$stock = 0;
-
-		$global_stock_enabled = get_post_meta( $post_id, Tribe__Tickets__Global_Stock::GLOBAL_STOCK_ENABLED, true ) == true;
-		$global_stock = $global_stock_enabled ?
-			get_post_meta( $post_id, Tribe__Tickets__Global_Stock::GLOBAL_STOCK_LEVEL, true )
-			: false;
+	protected function get_percentage_string( $post_id, array $deprecated = array() ) {
```
Comment (lucatume) [inline src/Tribe/Admin/Columns/Tickets.php]: Question: does it make sense to provide a default value, `array()`, for a deprecated value? Would it not be better to default to `null`?

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $deprecated is never used | [Tickets.php:117](https://github.com/moderntribe/event-tickets/blob/424ddf35ba495649ffa0760f923649249a265bb7/src/Tribe/Admin/Columns/Tickets.php#L117)

> via jenkins/codesniffer-phpcs

**Pull request merged**